feat: Added cluster overcommitment rate

dploeger opened the pull request:
This plugin calculates the rate between the actual existing CPUs and memory to the committed CPUs and memory in the virtual hardware of every VM.

Sneddo reviewed (commented):
Sorry it's been so long before reviewing this, this project has been stalled for a while. I'm back working with VMware after a bit of a break, so doing a bit of a cleanup.

Thanks for your contribution, the only suggestion I would make is that assigning $clusterInfo to the $restult variable is not requiredm it could just be sent to the output stream which is then captured by vCheck. This will be slightly more effficient.

Sneddo commented:
Secondly, PRs should be submitted to the dev branch- I'm not sure if this is actually stated anywhere, so easy to overlook.

dploeger commented:
@Sneddo Thanks! I have changed the base branch to dev and removed the result variable. I'm not that deep into Powershell. Could you take a look, if that is how it's supposed to be? Thanks.
